CMakeLists.txt

```diff
@@ -243,9 +243,12 @@ if(GPU)
 endif()
 
 # Add tests
-option(TESTS "Enable tests (needs Python and Numpy)" ON)
+option(TESTS "Enable tests (needs Python3 and Numpy)" ON)
 if(TESTS)
-  find_package (Python COMPONENTS Interpreter)
+  find_package (Python3 COMPONENTS Interpreter NumPy)
+  if(NOT Python3_FOUND)
+    message(FATAL_ERROR "Python3 and Numpy are required for MOPAC testing (testing can be disabled with -DTESTS=OFF)")
+  endif()
```

Comment:
Does it *really* need Python 3?

Reply:
Python 2 is basically considered obsolete at this point, right? I'm using several conveniences of Python 3 - formatted strings and the matrix multiplication symbol - and one essential feature - Python 3's subprocess.run can catch run-time errors as exceptions and let me process them within Python whereas Python 2's os.system does not seem to have comparable error-handling functionality.